docs/SNAPTELD_CONFIGURATION.md

```diff
@@ -17,11 +17,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
```

Comment:
Do we want to keep this name for this file? What about `docs/DAEMON_CONFIGURATION.md`?